docs/source/_templates/layout.html

```diff
@@ -121,13 +121,13 @@
           </li>
 
           <li>
-            <a href="{{ theme_variables.external_urls['home'] }}/master/about.html">About us</a>
+            <a href="{{ pathto('about') }}">About us</a>
```

Comment:
I think we should keep previous code as intention is to always show the recent info and not that one coined to the release...